src/test/java/org/broadinstitute/consent/http/AbstractTestHelper.java

```diff
@@ -5,6 +5,11 @@
 
 public abstract class AbstractTestHelper {
 
+  public static boolean disableTestContainers() {
+    String defaultProp = "false";
+    return System.getProperty("disableTestContainers", defaultProp).equals("true");
```

Comment:
`Boolean.parseBoolean()` is equivalent to this form; it handles `null` and returns `false` for non `"true"` values.
```suggestion
    return Boolean.parseBoolean(System.getProperty("disableTestContainers"));
```

Note that this maps misspelled boolean values to `false`. It may be better to fail in that case.

Reply:
@pshapiro4broad, Based on the plan for changing this to enableTestContainers, something like Boolean.parseBoolean(System.getProperty("enableTestContainers", "true"));
I'll also flip the conditionals in MockServerTestHelper and DAOTestHelper.
Reasonable?